README.md

```diff
@@ -46,7 +46,7 @@ console.log("Result: ", resultPmt);
 ```
 ### Tool Usage
 
-This library can be used via a tool that exists in the tool/pmt-builder.js file via the following command: `node tool/pmt-builder.js network blockHash txHash`, for example: `node tool/pmt-builder.js testnet 00000000000003d91235b675366fc6c26e0ea4b2f125fd292f164247d4e5b07e ac72bd61c72ac8143e5638998f479bdfc5834fce9576fa2054c7be93313abd66`
+This library can be used via a tool(this tool uses mempool.js api to get transactions) that exists in the tool/pmt-builder.js file via the following command: `node tool/pmt-builder.js network blockHash txHash`, for example: `node tool/pmt-builder.js testnet 00000000000003d91235b675366fc6c26e0ea4b2f125fd292f164247d4e5b07e ac72bd61c72ac8143e5638998f479bdfc5834fce9576fa2054c7be93313abd66`
```

Comment:
```suggestion
This library can be used via a tool(this tool uses mempool.js api to get transactions) that exists in the tool/pmt-builder.js file via the following command: 

`node tool/pmt-builder.js network blockHash txHash`

 For example: 
 
 `node tool/pmt-builder.js testnet 00000000000003d91235b675366fc6c26e0ea4b2f125fd292f164247d4e5b07e ac72bd61c72ac8143e5638998f479bdfc5834fce9576fa2054c7be93313abd66`
```

What do you think about put commands in separate lines from the text? I prefer to see commands in that way. Top me that way it is easier to read.

Reply:
Yeah, Resolved